neet/boolean/eca.py

```diff
@@ -365,10 +365,10 @@ def neighbors_out(self, index, *args, **kwargs):
 
         return {left, index, right}
 
-    def to_networkx_graph(self, *args, **kwargs):
+    def to_networkx_network(self, *args, **kwargs):
```

Comment:
The idea was to keep the naming consistent with the class that it's derived from. Technically the visualized graph/network as drawn also has a dynamical state provided by Neet, but we just don't show the states. I think keeping the naming consistent with the methods and how we describe the system (landscape graph vs network graph) is also important because drawing both the network and landscape is done through similarly named methods (ie `draw_network` and `draw_landscape`). Those draw methods are called directly on the network object itself--so to me, saying `s_pombe.draw_graph()` is ambiguous. Should I expect the network graph or the landscape graph? 

What are your thoughts?

Reply:
Right, but this isn't changing the name of the draw functions, it's changing the name of the function that yields a networkx graph. Right? What I'm saying is the following makes sense to me:
```python
s_pombe.to_networkx_graph()    # returns a networkx DiGraph
s_pombe.draw_network()         # draws the network
s_pombe.draw_landscape()       # draws the network's state transition gra[h
```

Another alternative is the following
```python
s_pombe.graph()                 # returns a networkx DiGraph
s_pombe.draw()                  # draws the network's graph
s_pombe.transition_graph        # the state transition graph as a DiGraph
s_pombe.draw_transition_graph() # you get the idea.
```

Thoughts?